supervisely/convert/image/yolo/yolo_converter.py

```diff
@@ -260,6 +261,11 @@ def to_supervisely(
                         )
                         if geometry is None:
                             continue
+                        if not img_rect.contains(geometry.to_bbox()):
+                            logger.warning(
+                                f"Annotation geometry is out of image bounds. Skipping..."
+                            )
+                            continue
```

Comment:
`get_geometry` uses relative yolo coordinates + image shape (no need to double-check bounds here)